src/Sentry.AspNetCore/SentryMiddleware.cs

```diff
@@ -139,12 +144,12 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
             // making the OnCompleted flush to not work.
             Task FlushBeforeCompleted() => hub.FlushAsync(timeout: _options.FlushTimeout);
 
-            void CaptureException(Exception e, SentryId eventId, string mechanism)
+            void CaptureException(Exception e, SentryId evtId, string mechanism)
             {
                 e.Data[Mechanism.HandledKey] = false;
                 e.Data[Mechanism.MechanismKey] = mechanism;
 
-                var evt = new SentryEvent(e, eventId: eventId);
```

Comment:
FWIW i prefer `eventId`

Reply:
Same, but there was a yellow warning due to hiding the outer variable with the same name.